app/src/utils/hooks.ts

```diff
@@ -132,3 +149,8 @@ export const useScenario = (scenarioId: string) => {
 };
 
 export const useVisibleScenarioIds = () => useScenarios().data?.scenarios.map((s) => s.id) ?? [];
+
+export const useSelectedOrg = () => {
```

Comment:
(it definitely would have been easier to rename org to project before writing all the additional code in this PR 😂)

Reply:
Yes, you're right. I've been thinking about orgs and projects for the last few days, and I think we may need both. Billing should probably be associated with an org, along with membership and privileges. However, I don't think we should invest time setting that up right now.